src/chroot.c

```diff
@@ -79,11 +80,12 @@ void chroot_create_full(int newversion)
 		}
 
 		LOG(NULL, "Recopy bundle chroot out of full", "%s", group);
-		string_or_die(&command, "rsync -aAX --existing %s %s/%i/%s",
-			      full_dir, image_dir, newversion, group);
-		ret = system(command);
-		assert(ret == 0);
-		free(command);
+		string_or_die(&param, "%s/%i/%s/", image_dir, newversion, group);
+		char *const rsynccmd[] = { "rsync", "-aAX", "--existing", full_dir, param, NULL };
+		if (system_argv(rsynccmd) != 0) {
+			assert(0);
+		}
+		free(param);
 	}
```

Comment:
I'm still seeing this code change here that won't apply against clearlinux/swupd-server master. Can you try the rebase again?